src/Components/TourGuideCard/tourGuideCard.css

```diff
@@ -0,0 +1,37 @@
+.cardSize {
+  margin: 80px;
```

Comment:
too much margin, just remove it you don't need it here, you can specify it later when putting this component in a list